StoreHouse/StoreHouse/ClientManagerAddOrderView.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections;
+using System.Runtime.Remoting.Messaging;
+using System.Windows.Forms;
+using Presentation;
+
+namespace View
+{
+    public partial class ClientManagerAddOrderView : Form, IClientManagerAddOrderView
+    {
+        private readonly ApplicationContext _context;
+
+        public event Action Back;
+        public event Action NextStep;
+
+        public ClientManagerAddOrderView(ApplicationContext context)
+        {
+            _context = context;
+            InitializeComponent();
+            this.StartPosition = FormStartPosition.CenterScreen;
+        }
+
+        private void radioBtn_Delivery_Clicked(object o, EventArgs e)
```

Comment:
please follow `PascaleCase` naming convention for methods. Note that method have to start from the verb: `AddCustomer`, `AddClientManagerOrderView` instead of `ClientManagerAddOrderView` and so on

Reply:
since this method is click handler rename it to `OnRadioBtnDeliveryClick`

Reply:
please do this refactoring for all classes in `View` module